ui/components/multichain/import-token-link/import-token-link.test.js

```diff
@@ -65,7 +66,7 @@ describe('Import Token Link', () => {
 
     const store = configureMockStore()(mockState);
 
-    renderWithProvider(<ImportTokenLink />, store);
+    renderWithProvider(<ImportTokenLink />, store); // should this be RefreshTokenLink?
```

Comment:
This component only contains `RefreshTokenLink` and the actual import button is move into the `asset-list-control-bar`. Should we rename this component and all the tests?

Reply:
I think so, since it only does refresh now

Reply:
I think I'm going to handle this in a separate PR to keep the diff low, and this doesn't affect functionality.

https://consensyssoftware.atlassian.net/browse/MMASSETS-416